fcd_weight_scale_mrp/models/fcd_chkpoint.py

```diff
@@ -15,6 +15,8 @@ class FCDCheckpoint(models.Model):
     warehouse_id = fields.Many2one('stock.warehouse')
     printer_ip = fields.Char()
     printer_port = fields.Integer(default=21)
+    printer_user = fields.Char()
+    printer_pass = fields.Char()
```

Comment:
These fields should be required, or `fcd_weight_scale_log.py` comms command should become optional.